orttraining/orttraining/training_ops/cuda/communication/send.h

```diff
@@ -21,6 +22,20 @@ class Send final : public CudaKernel {
   Status ComputeInternal(OpKernelContext* context) const override;
 
 private:
+  void SendShapeInfo(
+      const int dst,
```

Comment:
>dst [](start = 16, length = 3)

could use a more descriptive variable name. It's hard to tell what dst mean from the interface...